### PR TITLE
fix(docs): Remove `config.toml` remnants

### LINKED ISSUE #1126: Remove reference to `config.toml` from documentation
### Bug description

We still have some docs that mention the (removed) `config.toml`.

### PR DESCRIPTION
**Pull request by Alex6323**
## Linked Issues

* Closes #1126

### CODE REVIEW

File: documentation/docs/reference/authentication.md [hunk 1]
```diff
@@ -9,22 +9,7 @@ keywords:
...
-
-The `[api.argon_config]` sub-table can be used to configure the argon2 hash generation, using the following settings:
-
-- `hash_length` - The length of the resulting hash in bytes.
-- `parallelism` - The number of threads used.
-- `mem_cost` - The memory used.
-- `iterations` - The number of iterations to perform.
-- `variant` - One of three variants: [argon2d](https://www.argon2d.com/), [argon2i](https://www.argon2i.com/), or [argon2id](https://www.argon2id.com/).
-- `version` - The argon version (currently 0x13).
+Usage of the Chronicle API can be protected using [JWT](https://jwt.io/) by using Chronicle's command-line interface (CLI). Run `inx-chronicle --help` and read about all available options under the `API:` section.
```
Comment (Alexandcoats) [inline documentation/docs/reference/authentication.md:12]: Maybe just `API` without the colon here.

File: documentation/docs/reference/environment.md [hunk 2]
```diff
@@ -10,7 +10,9 @@ keywords:
...
-* `CONFIG_PATH="<FILE_PATH>"`: sets the file path to the `config.toml` file;
-* `INX=<true|false>`: enables/disables INX;
-* `API=<true|false>`: enables/disables the REST API;
-* `METRICS=<true|false>`: enables/disables the Metrics Server;
+* `MONGODB_CONN_STR`: sets the MongoDb connection string including credentials;
+* `INFLUXDB_USERNAME`: sets the InfluxDb user;
+* `INFLUXDB_PASSWORD`: sets the InfluxDb password;
+* `JWT_IDENTITY`: sets the filepath to a JWT identity file;
+* `JWT_PASSWORD`: sets the JWT password;
+* `JWT_SALT`: sets the JWT salt;
```
Comment (Alexandcoats) [inline documentation/docs/reference/environment.md:18]: There are a few more now, like `INX_URL`.

Comment (Alex6323) [reply documentation/docs/reference/environment.md:18]: yeah, sorry ... I branched off of `main`.

Comment (Alexandcoats) [reply documentation/docs/reference/environment.md:18]: Right, I forgot. Well once we merge this we should create a new PR on the feature branch to add the new stuff.

**Review by Alexandcoats — APPROVED** [review]

**Pull request merged**